tests/NuGet.Services.BasicSearchTests/NuGet.Services.BasicSearchTests.csproj

```diff
@@ -112,9 +112,6 @@
     <PackageReference Include="Lucene.Net">
       <Version>3.0.3</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.ApplicationInsights">
```

Comment:
Transitive dependency of NuGet.Services.Logging. Upgraded to version 2.10.0 as a result.